test/nodejs/Libsvm_Svctest.js

```diff
@@ -445,193 +574,7 @@ describe("LIBSVM SVC test", function () {
             var SVC2 = new analytics.SVC(require('qminer').fs.openRead('svr_test.bin'));
             assert.deepEqual(SVC.getParams(), SVC2.getParams());
             assert.eqtol(SVC.weights.minus(SVC2.weights).norm(), 0, 1e-8);
-            assert.eqtol(Math.abs(SVC.bias - SVC2.bias), 0, 1e-8);
         })
     });
-    describe('Fitting tests', function () {
-        it('should fit a model on iris dataset (dense), class=setosa, features:sepal length, sepal width', function () {
```

Comment:
Why were this tests deleted? It appears as a bad merge. Please be careful not to override recent changes in the code when merging.